.vscode/launch.json

```diff
@@ -0,0 +1,54 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"name": "Unity Editor",
+			"type": "unity",
+			"path": "/Users/gordie/Documents/GitHub/torte/Library/EditorInstance.json",
```

Comment:
We probably don't want VS code settings here? but especially with references to a user's filesystem

Reply:
Whoops, that's what i get for staging my entire working directory. This is a runner requied to use the unity debugger, perhaps it's best added to the ignore.